environment/env.go

```diff
@@ -26,13 +26,16 @@ package environment
 
 import (
 	"fmt"
+	"net"
 	"os"
 	"strconv"
 )
 
 const (
-	// Localhost default localhost
-	Localhost = "127.0.0.1"
```

Comment:
effectively `Localhost` is removed in favor of requiring a call to `GetLocalhostIP()` and we're also adding an env var for LOCALHOST_IP which feels a little odd but being explicit about which localhost ip address is required in dual stack ipv4/ipv6 envrionments because golang wil always choose ipv4 by default.